packages/closer/pages/events/[slug]/report.tsx

```
@@ -0,0 +1,330 @@
+    }
+
+    // Aggregate tickets by type
+    if (ticketTypeMap.has(ticketType)) {
+      ticketTypeMap.set(ticketType, {
+        count: ticketTypeMap.get(ticketType).count + quantity,
+        revenue: ticketTypeMap.get(ticketType).revenue + price
+      });
+    } else {
+      ticketTypeMap.set(ticketType, { count: quantity, revenue: price });
+    }
+  });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)